### ISSUE TITLE
Multiline f-strings not foldable

### ISSUE THREAD

**Issue by headtr1ck** [microsoft/pylance-release#2271]
VS Code does not recognize multiline f-strings as foldable.

Example:
```
f = """Lorem ipsum dolor sit amet, consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua.
    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat.
    Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
    Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
"""
f2 = f"""Lorem ipsum dolor sit amet, {1} consectetur adipisici elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua.
    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat.
    Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
    Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
"""
```
`f` is foldable, `f2` is not, that is the triangle between line number and code is not showing up and "Fold All" does not collapse it.

Not sure if this is a python extension problem or a VS Code problem.

**Comment by debonte** [comment]
This will be fixed in the next release of Pylance.

**Comment by bschnurr** [comment]
This issue has been fixed in version 2022.1.3, which we've just released. You can find the changelog here:  [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202213-21-january-2022)
